export const getProductOneResolver = async (parent, args, context) => {
    try {
        console.log("Reached here on the resolver")
    } catch (error) {
        
    }
}
